geoportailv3/static/js/draw/drawnfeaturesservice.js

```diff
@@ -121,11 +121,10 @@ app.DrawnFeatures.prototype.encodeFeaturesInUrl_ = function(features) {
 /**
  * Save the feature either in url or in mymaps if the current user
  * has the permissions.
- * @param {ol.Feature} feature the feature to save.
+ * @param {ol.Feature} feature The feature to save.
  */
 app.DrawnFeatures.prototype.saveFeature = function(feature) {
-  if (this.appMymaps_.isEditable()) {
-    feature.set('__source__', 'mymaps');
+  if (this.appMymaps_.isEditable() && feature.get('__source__') == 'mymaps') {
```

Comment:
This is very minor but you changed `!=` to `!==` above, and here you use `==` rather than `===`. You could be more consistent.